### ISSUE TITLE
Repeated notifications for one event

### ISSUE THREAD

**Issue by rmenessec** [indywidualny/FaceSlim#146]
FaceSlim is throwing multiple notifications for a single event on the Facebook side. What's more, it's doing it even if I mark all notifications read through the desktop version of the site.

Speaking of which, FaceSlim could really use some equivalent of "mark all read", since the mobile site doesn't appear to provide it...

**Comment by indywidualny** [comment]
It checks by date. If for example someone posted again below the same post you will be notified again. I thought it's a nice behaviour. When a date of a notification is updated by Facebook you are notified again. It ignores marking posts read. Perhaps it should be redesigned but I'm gonna leave it for now. As I said, if there is something newer than the date saved during the last check the notification will appear.

**Comment by rmenessec** [comment]
I'm getting re-notified for at least some events that show no signs of having been updated. For instance, I'll get a notification that someone has commented on one of my posts. Then it will happen again, even though the original comment has not changed and no one else has commented. I'll get three or four or five notifications, even though nothing appears to have been updated.

So far, all the bogus extra notifications appear to be for comments or for likes on my posts, or people commenting on a post after I've commented. Notifications for new messages are working correctly.

Could this be caused by the page cache?
